### PR TITLE
Update IRIS DRS to 2021.04.08

### PR DESCRIPTION
**Pull request by zonca**
* Rework of the TCS telemetry
* FITS keywords into the usage field

Human readable csvdiff:

https://github.com/oirlab/iris_metadata/blob/main/csvdiff/2021.04.08.txt
https://github.com/oirlab/iris_metadata/blob/main/csvdiff/2021.04.05.txt
https://github.com/oirlab/iris_metadata/blob/main/csvdiff/2021.03.24.txt

### CODE REVIEW

File: drs/drs-assembly/subscribe-model.conf [hunk 1]
```diff
@@ -313,17 +369,44 @@ IRIS, TCS, ESW, NFIRASO, AOESW and NSCU to be associated with science data
     {
       subsystem = "IRIS"
       component = "sc.cryoenv"
-      name = "IMGTEMP[XX]"
+      name = "imgDetectorTemperature"
```
Comment (edwardchapin) [inline drs/drs-assembly/subscribe-model.conf:372]: I did a test ingest of these changes and took a look with icdwebserver. I noticed that this block of events that you subscribe to from the sc.cryoenv assembly appear not to be published. Just curious if these are proposed changes and you will ask the owner of that assembly to produce the requested items?

**Review by edwardchapin — APPROVED** [review]
All of the external interfaces look good. Only issue I noticed was the noted problem with sc.cryoenv. Presumably you will follow-up with Ji Man, but I see no reason to hold up merging this branch at this point if you want to go ahead.

**Comment by zonca** [comment]
@arunsurya77 can you please check this?

File: drs/drs-assembly/subscribe-model.conf (hunk 1 above)
Comment (tnakamot) [reply drs/drs-assembly/subscribe-model.conf:372]: @jimansohn can chime in, but those items are defined in his working branch:
https://github.com/tmt-icd/IRIS-Model-Files/blob/sohn_cryoUpdate/sc/cryoenv-assembly/publish-model.conf

Comment (arunsurya77) [reply drs/drs-assembly/subscribe-model.conf:372]: Yes these changes reflect the updates on Ji Man's branch recently.  I think the current pull request looks fine.

Comment (jimansohn) [reply drs/drs-assembly/subscribe-model.conf:372]: I created PR #37 to avoid the error when the master is ingested.

**Pull request merged**